test/integration/getTags.test.ts

```diff
@@ -0,0 +1,100 @@
+import * as contentful from '../../lib/contentful'
+import { params } from './utils'
+
+if (process.env.API_INTEGRATION_TESTS) {
+  params.host = '127.0.0.1:5000'
+  params.insecure = true
+}
+
+const client = contentful.createClient(params)
+
+describe.only('getTags', () => {
+  it('returns all tags when no filters are available', async () => {
+    const response = await client.getTags()
+
+    console.dir(response, { depth: null })
+
+    expect(response.items[0].sys.type).toBe('Tag')
+  })
+
+  describe('tagName filters', () => {
+    it('gets the tag with the name equals to the provided value', async () => {
+      const response = await client.getTags({ name: 'public tag 1' })
+
+      expect(response.items.length).toBe(1)
```

Comment:
Nitpick: if you phrase the assertion to be `expect(response.items).toHaveLength(1)` it will have more human-friendly error message in case it fails ("expected 2 to be 1" vs "expected length: 1, received length: 2").

Reply:
Noted, I will start using that from now on